eap_backend/eap_api/urls.py

```diff
@@ -21,11 +21,17 @@
         views.property_claim_list,
         name="property_claim_list",
     ),
+    # path(
+    #     "comments/<int:assurance_case_id>/",
+    #     views.comment_list,
+    #     name="comment_list",
+    # ),
     path(
-        "comments/<int:assurance_case_id>/",
+        "assurance_case/<int:assurance_case_id>/comments/",
```

Comment:
Although I might disagree with the initial API design, I see it is important to keep in consistent with all the other resources: `case_element_in_plural/` for listing/creation deletion where you can send the `assurance_case_id` via request parameters instead of via slug. Please adjust accordingly.

Reply:
@cptanalatriste do you mean changing `assurance_case` to `assurance_cases` ?

Reply:
I was thinking doing something similar to `goals`, another top-level case item attached to assurance case (see `goal_list` and `goal_detail`). In that style, we would have two resources: `comments/` to handle case-level retrieve and creation and `comments/<int:pk>` for update, deletion and single-comment retrieval. Just to keep the same design over the whole API